torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -457,36 +459,60 @@ graph(%input, %weight, %bias, %4):
      %second_output = aten::add_(%first_output, %bias, %4)
      return (%second_output) )");
 
-  const PatternInfo add_nn_relu = PatternInfo::parse_from_str(R"(
-graph(%self, %a, %b):
-     %one = prim::Constant[value=1]()
-     %first_output = aten::add(%a, %b, %one)
-     %second_module = match::module[name="ReLU"](%self)
-     %second_output = prim::CallMethod[name="forward"](%second_module, %first_output)
+  const PatternInfo add_nn_relu = PatternInfo::parse_from_str(
+      R"(
+graph(%self, %a, %b, %alpha, %relu):
+     %first_output = aten::add(%a, %b, %alpha)
+     %second_output = prim::CallMethod[name="forward.*"](%relu, %first_output)
```

Comment:
Is there a convention that the callmethod always has the name of the form forward.*?

Reply:
this is to handle methods produced by tracing, I'll probably use forward followed by digits to be more accurate

Reply:
callmethod can have any name.